e2e/specs/program-enrollment.spec.ts

```diff
@@ -71,7 +71,7 @@ test('Add and edit a program enrollment', async ({ page, api }) => {
   await test.step('Then I should see the updated program enrollment in the list', async () => {
     await expect(dataRow).toContainText(/hiv care and treatment/i);
     await expect(dataRow).toContainText(/03-Jul-2023/i);
-    await expect(dataRow).toContainText(/completed on 04-Jul-2023/i);
+    await expect(dataRow).toContainText(/completed on 05-Jul-2023/i);
```

Comment:
@denniskigen the issue is somewhere else. Actually the issue happens when editing the program enrollment.

Reply:
I should be in position to send a commit in the next 10 minutes that fixes the bug.